code/style.css

```diff
@@ -1,4 +1,308 @@
-html {
-  background: #0025ff;
-  color: #fff;
+
+  html {
+  background-image: url(./images/background.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  color: rgb(0, 0, 0);
+  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
 }
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+header { 
+  background-color: rgba(255, 255, 255, 0.747);
+  height: 80px;
+  width: 380px;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: left;
+  margin-top: 10px;
+}
+
+h1 {
+  color: rgb(0, 0, 0);
+  font-size: 28px;
+  margin-left: 75px;
+  padding: 10px;
+  padding-right: 5px;
+  text-shadow: 2px 2px 5px rgb(163, 163, 163);
+}
+
+h2 {
+  font-size: 20px;
+  padding: 0;
+  padding-bottom: 1px;
+  margin: 8px;
+  margin-bottom: -8px;
+}
+
+h3 {
+  font-size: 18px;
+  padding: 4px; margin: 0;
+  padding-top: 10px;
+}
+
+p {
+  font-size: 16px;
+  padding: 8px;  
+}
+
+hr {
+  width: 100%;
+}
+
+.hamburger {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  width: 20px;
+  height: 20px;
+  margin-left: 15px;
+}
+
+.hamburger div {
+  width: 20px;
+  height: 3px;
+  background-color: rgb(65, 211, 155);
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.left, .right {
+  color: #000;
+  font-size: 16px;
+  text-decoration: none;
+  display: none;
+}
+
+.left-links, .right-links {
+  color:rgb(65, 211, 155);
+  text-decoration: none;
+}
+
+.main {
+  background-color:rgba(255, 255, 255, 0.747);
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  height: 450px;
+  width: 380px;
+  margin-bottom: 30px;
+}
+
+.main a:link {
+  color: #000;
+  font-size: 16px;
+  text-decoration: none;
+}
+
+.main a:hover {
+  color: rgb(65, 211, 155);
+}
+
+.main a.visited {
+  color: grey;
+}
+
+#mainphoto {  
+  width: 92%;
+  height: auto;
+  margin: 10px;
+  padding: 5px;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+img {
+  width: 300px;
+  max-width: 100%;
+  height: 400px;
+  display: block; 
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19)
+}
+
+.cards {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 5px;
+ }
+
+ .card {
+  background-color: rgba(255, 255, 255, 0.747);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 10px;
+  padding-top: 15px;
+  margin: 20px;
+  margin-bottom: 2px;
+  width: 350px;
+  height: 450px;
```

Comment:
If you want the cards to fit on a mobile you could use the media queries and a with in % instead of pixels. Doesn't require changes right now but something to remember as you continue 😊